app/experimenter/experiments/migrations/0221_nimbusexperiment_prevent_pref_conflicts.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.16 on 2022-11-01 00:39
```

Comment:
Oh yeah fold these two together, you can just delete them and rerun `make makemigrations` and it'll smush them together.